crim/admin.py

```diff
@@ -682,6 +682,7 @@ class UserAdmin(UserAdmin):
 
 admin.site.unregister(User)
 admin.site.register(User, UserAdmin)
+admin.site.register(CRIMUserProfile)
```

Comment:
We don't want to add `admin.site.register(CRIMUserProfile)` -- our custom user profile is automatically linked to the main user account, and the additional details can be updated there. Perhaps we can add a comment explaining this.